src/client/opamCommands.ml

```diff
@@ -624,6 +624,33 @@ let show =
            $file $normalise $atom_or_local_list)),
   term_info "show" ~doc ~man
 
+module Common_config_flags = struct
+  let sexp =
+    mk_flag ["sexp"]
+      "Print environment variables as an s-expression rather than in shell \
+       format"
+
+  let inplace_path =
+    mk_flag ["inplace-path"]
+      "When updating the $(i,PATH) variable, replace any pre-existing opam \
+       path in-place rather than putting the new path in front. This means \
+       programs installed in opam that were shadowed will remain so after \
+       $(b,opam config env)"
+
+  let set_opamroot =
+    mk_flag ["set-root"]
+      "With the $(b,env) and $(b,exec) subcommands, also sets the \
+       $(i,OPAMROOT) variable, making sure further calls to opam will use the \
+       samme root."
+
+  let set_opamswitch =
+    mk_flag ["set-switch"]
+      "With the $(b,env) and $(b,exec) subcommands, also sets the \
+       $(i,OPAMSWITCH) variable, making sure further calls to opam will use \
+       the same switch. This can be used to select a local switch from a \
```

Comment:
It's a bit unclear to what exactly the `OPAMSWITCH` variable is going to be set. I.e. what the "same switch" is. Also I would drop the final sentence or extend it, a switch is not necessarily determined from the `cwd` e.g. it could be determined from an existing `OPAMSWITCH`.